### DISCUSSION TITLE
Check the sharing email by default

### DISCUSSION THREAD

**Discussion by Misa898** [aboutyou/dart_packages#392] [Q&A] [answered]
Hey there. Is there any way to check the "Share my email" by default
![Uploading Capture d’écran 2023-12-20 à 14.39.33.png…]()

**Comment by tp** [comment, marked as answer]
No, there are no parameters we can pass to this piece of native UI (https://support.apple.com/en-us/102379).

Having this would of course be very beneficial for the user if they already have an account with that e-mail which you could then merge in the backend to also allow SiwA.
But alas for new sign-ups it's probably the better choice to not share the personal e-mail address, thus I understand why this would be the default.
